modules/3d-tiles/src/lib/parsers/helpers/parse-3d-tile-subtree.ts

```diff
@@ -42,29 +42,49 @@ export default async function parse3DTilesSubtree(data: ArrayBuffer): Promise<Su
     subtree.tileAvailability.explicitBitstream = await getExplicitBitstream(
       subtree,
       'tileAvailability',
-      internalBinaryBuffer
+      internalBinaryBuffer,
+      options.basePath
     );
   }
 
   if ('bufferView' in subtree.contentAvailability) {
     subtree.contentAvailability.explicitBitstream = await getExplicitBitstream(
       subtree,
       'contentAvailability',
-      internalBinaryBuffer
+      internalBinaryBuffer,
+      options.basePath
     );
   }
 
   if ('bufferView' in subtree.childSubtreeAvailability) {
     subtree.childSubtreeAvailability.explicitBitstream = await getExplicitBitstream(
       subtree,
       'childSubtreeAvailability',
-      internalBinaryBuffer
+      internalBinaryBuffer,
+      options.basePath
     );
   }
 
   return subtree;
 }
 
+function resolveUri(uri, basePath) {
```

Comment:
Parameter types and return values, JSDoc

Reply:
done